RELEASE.md

```diff
@@ -8,16 +8,15 @@ Please follow the established format:
 # Release 9.2.0
 
 ## Major features and improvements
+
 - Enable/disable preview for all the datasets when publishing Kedro-Viz from CLI. (#1894)
 - Enable/disable preview for all the datasets when publishing Kedro-Viz from UI. (#1895)
-
-## Bug fixes and other changes
-
-## Major features and improvements
 - Display published URLs. (#1907)
 
 ## Bug fixes and other changes
 
+- Fix numpy 2.0 e2e test fail with lower requirements. (#1949)
```

Comment:
Doesn't need to be in the release notes I think as it's not a user facing change!

Reply:
I am not sure on this. We do mention changes in the - ## Bug fixes and other changes section which are not user facing (like - - Migrate from CircleCi to GitHub Actions. (#1876)). But will remove it for this PR. Need to know what to mention in this section of the release note.